redisdb/datadog_checks/redisdb/redisdb.py

```diff
@@ -449,7 +449,8 @@ def _check_slowlog(self):
 
         # Get all slowlog entries
 
-        slowlogs = conn.slowlog_get(max_slow_entries)
+        # https://github.com/andymccurdy/redis-py/issues/1238
+        slowlogs = conn.slowlog_get(max_slow_entries, decode_responses=False)
```

Comment:
Do you mean to use `parse_slowlog_get`? https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L401

Reply:
I was looking at https://github.com/andymccurdy/redis-py/pull/1239
but seems that was changed later

Reply:
Oh, yes, indeed, this methods uses the connection options, and 'decode_responses' is false by default https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L744